### PR TITLE
Added Graviton3 as target architecture

### PR DESCRIPTION
**Pull request by OliverPerks**
I think I have a 'good' set of architecture flags for the different compilers. Some may be missing, happy to iterate if we want to check any.

### CODE REVIEW

**Comment by OliverPerks** [comment]
Sorry, only just seen #51 - This one adds the 'correct' flags.

**Comment by boegel** [comment]
This looks way more mature than my PR #51, so I'll close that.

@alalazo Do we have a procedure in place to check/accept PRs like this?

**Review by alalazo — CHANGES_REQUESTED** [review]
Can you add a sample of `/proc/cpuinfo` in the `tests/targets` directory? That will allow us to test detection of `graviton3`

**Comment by OliverPerks** [comment]
@alalazo - Sorry forgot to add that file. Done now.

File: cpu/microarchitectures.json [hunk 1]
```diff
@@ -2280,6 +2280,107 @@
           ]
       }
     },
+    "graviton3": {
+      "from": ["aarch64"],
+      "vendor": "ARM",
+      "features": [
+	        "fp",
+          "asimd",
+          "evtstrm",
```
Comment (alalazo) [inline cpu/microarchitectures.json:2289]:
```suggestion
	  "fp",
          "asimd",
          "evtstrm",
```

**Review by alalazo — APPROVED** [review]
LGTM. Just want to double check one thing: is it correct to root each `gravitonX` to `aarch64` instead of having:
```
aarch64 -> graviton -> graviton2 -> graviton3
```
? I think the question is, can we say that a binary optimized for `gravitonX` can always run on `gravitonY` fot `X <= Y`?

**Comment by OliverPerks** [comment]
That is a good question. In theory Graviton X could have a feature Graviton X+1 doesn't support, but in practice they are strict subsets.

**Review by alalazo — APPROVED** [review]

**Comment by alalazo** [comment]
Thanks @OliverPerks

**Pull request merged**